docs/appendices/release-notes/5.9.0.rst

```diff
@@ -45,6 +45,10 @@ Version 5.9.0 - Unreleased
 Breaking Changes
 ================
 
+- Fixed an issue that caused ``WHERE`` clause containing ``ARRAY_LENGTH` scalar
+  function to incorrectly filter array of nulls. It is a breaking change
+  because the fix causes performance degradations.
```

Comment:
I think the posted numbers make the scalar pretty much unusable on bigger datasets. Is there an alternative we can recommend? 

Does wrapping it in ignore_3vl work? 

If not we need to come up with an alternative

Reply:
e.g. `where ignore3vl(array_length(xs) > 1)`

Although the semantic would be a bit differently to what we currently document for it. So far it is about the column being null, not elements of it.